docs/src/pages/components/slider-styled/UnstyledSlider.tsx

```diff
@@ -55,9 +56,22 @@ const StyledSlider = styled(SliderUnstyled)`
       right: -15px;
       bottom: -15px;
     }
+
+    :hover,
+    &.Mui-focusVisible {
+      box-shadow: 0 0 0 8px ${alpha('#000', 0.16)};
+    }
+
+    &.mui-active: {
```

Comment:
`yarn prettier` is changing this line :( 

```diff
-    &.mui-active: {
+    &.Mui-active: {
```

Reply:
I somewhat welcome the change. classnames are case insensitive anyway so having prettier format them doesn't change any runtime behavior.